internal/adapter/http/user.go

```diff
@@ -17,6 +17,15 @@ func NewUserController(usecase interfaces.User) *UserController {
 	}
 }
 
+type PasswordResetRequestInput struct {
+	Email *string `json:"email"`
```

Comment:
```suggestion
	Email string `json:"email"`
```

Email is required, so it does not need to be pointer